fury/actor.py

```diff
@@ -2740,3 +2742,92 @@ def callback(
                     block="light")
 
     return sq_actor
+
+
+def bitmap_labels(
```

Comment:
Maybe `billboard_labels` is more suitable for this actor. 
@skoudoro @filipinascimento @Garyfallidis

Reply:
Just  `actor.labels`? I don't like `bitmap_labels` because in the future we can use the same actor to draw text using msdf which has more than one channel and is not a bitmap.

Reply:
I would call it `styled_text`.

Reply:
final decision meeting (09/30/2021): 
- The name will be `label_fast`
- we need to rename `label` to `vector_text`
- After a release, `label_fast` will become `label`